Fix broken how to link for Kubernetes How to

chrisreddington opened the pull request:
Adding an "s" onto the end of the URL path for the k8s how to, as it is currently broken.

azurecla commented:
Hi __@sahaj08__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (sahajs). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

rmca14 commented:
@barbkess

chrisreddington committed:
Fix broken how to link for Kubernetes How to

Adding an "s" onto the end of the URL path for the k8s how to, as it is currently broken.
